operator/src/main/helm/coherence-operator/dashboards/cache-details-dashboard.json

```diff
@@ -777,7 +777,7 @@
           "decimals": 2,
           "link": true,
           "linkTooltip": "Drill down to member details for  ${__cell}",
-          "linkUrl": "/d/coh-member-details?refresh=10s&orgId=1var-ClusterName=$ClusterName&var-MemberName=${__cell}",
```

Comment:
Just curious why orgId was removed.

Reply:
OrgId should be removed as it should not be manually set in-case customer wishes to add dashboards in different org. By default the UI will automatically set the orgId, but if left out it will default to the org that the dashboards where imported into, which is correct behaviour.